scripts/install-devnet.sh

```diff
@@ -2,4 +2,4 @@
 
 set -e
 
-pip3 install starknet-devnet==0.1.22
+pip3 install starknet-devnet==$(../scripts/get-version.py starknet-devnet)
```

Comment:
It would cleaner if version was in a variable

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/95/commits/3b668f233ce0ad839e9f0994f922f53d497ba3e6